manifest.outlook.xml

```diff
@@ -32,79 +34,67 @@
     </Form>
   </FormSettings>
   <Permissions>ReadWriteItem</Permissions>
+  <!-- The <Rule> element is required for validation, but is ignored when there is a <VersionOverrides> element. -->
   <Rule xsi:type="RuleCollection" Mode="Or">
     <Rule xsi:type="ItemIs" ItemType="Message" FormType="Read" />
   </Rule>
-  <DisableEntityHighlighting>false</DisableEntityHighlighting>
   <VersionOverrides xmlns="http://schemas.microsoft.com/office/mailappversionoverrides" xsi:type="VersionOverridesV1_0">
-    <Requirements>
-      <bt:Sets DefaultMinVersion="1.3">
-        <bt:Set Name="Mailbox" />
-      </bt:Sets>
-    </Requirements>
+    <!-- The <Hosts> element is required for validation, but it is ignored when there there is a child <VersionOverrides> element of type VersionOverridesV1_1 with its own <Hosts> element. -->
     <Hosts>
       <Host xsi:type="MailHost">
-        <DesktopFormFactor>
```

Comment:
I don't think I understand what you are asking here. I've removed the Commands stuff from all of the manifests, not just Outlook. I've kept the Open Taskpane button in all of them, including Outlook. Outlook is not being treated differently, except that Outlook supports having a v. 1.1 VersionOverrides element as a child to the v. 1.0 VersionOverrides, whereas WXP do not. Because all recent Outlook features use that inner VersionOverrides, I've updated the Outlook manifest to use it.

Reply:
Yes . . . you are treating all of them the same with this change.  Whereas before it, Outlook was the only one that actually had a command entry point in the ribbon and the other just had some information in the manifest, you remove all uses of that.  Replicating this across the templates would remove commands from our teamplates.

I'm suggesting that instead of removing it everywhere, you add the ribbon entry point to the other host to include it everywhere.

Reply:
I see. I thought of that, but I decided not to go down that road. A function command (ribbon button that invokes a command) is just one of many ways that a task pane add-in can be enhanced. It seems arbitrary to me to include that particular enhancement and none of the others. It's true that it would make things easier for devs who want to add a function command, but it makes it harder for those who don't, since the latter have to (or should, at least) delete all that code and markup. Since these projects are not meant primarily to educate, but to be the starting place of a new add-in, I think it's best to keep them minimal.

Reply:
It's not really an enhancement of a taskpane add-in, but another shape an add-in can take (i.e. an add-in can be just a command without a taskpane).  I agree that the templates should be targeted and not necessarily have everything all together.  On the other hand, if we remove the command from the taskpane templates then we should have a template that is for commands.  The side effect of that is another repo to manage and that will be one more thing that customers will ask for a combination to start from (We already get request for things like a template that include react + custom functions or react + sso).  So somewhere we need to strike a balance many targeted starting points and a combined starting point.

Personally I keep thinking that a reworking of our templates and generator such that we provide the option for targeted and combination of features (also reducing the repos that need maintenance) would be nice, but that's not in the plans right now.  In the meantime, I think we should keep what we have until we have a specific plan to replace it (which doen't appear to be the case here).

Reply:
You've actually made me more convinced that it's just arbitrary to include a function command and not other things (whether they are called enhancements or shapes). And, as you say, we can't include everything. I also like the idea of having a tool where the user can select combinations of features. We can make that a North Star goal. But the React templates are out-of-date now and we can't hold up this PR until we reach the North Star. I don't think we need to replace the function commands in this PR any more than we need to add support for custom functions or keyboard shortcuts or SSO in this PR. 

At any rate, since my team owns the templates now, I think it is our call. I will bring up the question of whether to add back a function command to these templates in one of our team meetings in October. In the meantime, I'd like this PR to be merged without the function command for now.

Reply:
I've put the commands stuff back in.